package/src/components/MessageInput/FileUploadPreview.tsx

```diff
@@ -235,11 +286,12 @@ export const FileUploadPreview = <
   props: FileUploadPreviewProps<StreamChatGenerics>,
 ) => {
   const { fileUploads, removeFile, uploadFile } = useMessageInputContext<StreamChatGenerics>();
+  const { appSettings } = useChatContext<StreamChatGenerics>();
   const { FileAttachmentIcon } = useMessagesContext<StreamChatGenerics>();
 
   return (
     <MemoizedFileUploadPreview
-      {...{ FileAttachmentIcon, fileUploads, removeFile, uploadFile }}
+      {...{ appSettings, FileAttachmentIcon, fileUploads, removeFile, uploadFile }}
```

Comment:
I am looking at it now and I am beginning to think that the appSettings does not have to be in this component and this might be a mistake I missed while rectifying in the past.

Reply:
I'll be removing appSettings from this component as it's not performing any function and could lead to bugs in the future is left here. Thanks for catching the code smell @santhoshvai